### PR TITLE
added listener + logic to Webclient Save Layout button -  #2247

### PR DESCRIPTION
**Pull request by johnnyvoruz**
#2247 

**Brief overview of PR changes/additions**

- Introduced a jQuery listener on the #layoutName input field.
- Automatically disables the "Save" button (class .savelayout) if the input is exactly "default".
- Re-enables the "Save" button for other input values.
- This change ensures that users don't have the perception that they can overwrite the default layout.

**Motivation for adding to Evennia**

- I'm still learning the engine, but excited to contribute to the web site of things!

**Other info (issues closed, discussion etc)**

- This change was inspired by an observed behavior where users would often mistakenly try to save a new layout as "default".
- By disabling the button for this specific input, we add a subtle hint to the user about the importance of this default name and guide them towards choosing a different name.

### CODE REVIEW

File: evennia/web/static/webclient/js/plugins/goldenlayout.js [hunk 1]
```diff
@@ -584,12 +587,33 @@ let goldenlayout = (function () {
     }
 
 
-    //
+    // Listener for realtime changes to the layout name input field.
+    // If the layout name is "default", the save button is disabled
+    // to prevent the perception of overwriting the default layout.
+    $(document).on("input", "#layoutName", function () {
+        console.log("Input changed to:", $(this).val());
```
Comment (Griatch) [inline evennia/web/static/webclient/js/plugins/goldenlayout.js:594]: This seems like a leftover debug message (no need to log every change like this, right?)

**Review by Griatch — CHANGES_REQUESTED** [review]
Nice! Good to get this fixed. Just a minor tweak needed before merge.

File: evennia/web/static/webclient/js/plugins/goldenlayout.js (hunk 1 above)
Comment (johnnyvoruz) [reply evennia/web/static/webclient/js/plugins/goldenlayout.js:594]: Bingo, thanks for catching that @Griatch - that was just for debugging! Thanks for your eagle eyes!

**Review by Griatch — APPROVED** [review]
Looking good!

**Pull request merged**